src/components/TransactionsTable/index.tsx

```diff
@@ -134,12 +137,42 @@ export class TransactionsTable extends React.Component<TransactionsTableProps> {
               <Table responsive>
                 <thead>
                   <tr>
-                    <th className="transactions-table__asset-col">Asset</th>
-                    <th className="transactions-table__date-col">Date</th>
-                    <th className="transactions-table__operation-col">
+                    <th
+                      className="transactions-table__asset-col"
+                      // tslint:disable-next-line:jsx-no-lambda
+                      onClick={() =>
+                        this.props.handleColumnHeaderClick &&
```

Comment:
I think it make sense to move click handlers to separate function and check that handleColumnHeaderClick is presented inside. This will allow us to prevent copy-paste for all click handlers

Reply:
Good catch, fixed